presto-product-tests/src/main/java/io/prestosql/tests/hive/TestExternalHiveTable.java

```diff
@@ -46,6 +53,27 @@ public Requirement getRequirements(Configuration configuration)
                 mutableTable(NATION_PARTITIONED_BY_BIGINT_REGIONKEY));
     }
 
+    @Test
+    public void testCreateExternalTableWithInaccessibleSchemaLocation()
```

Comment:
this fails on CI

```
2020-07-11T18:47:14.9602286Z tests               | io.prestosql.tempto.query.QueryExecutionException: java.sql.SQLException: Query failed (#20200711_184714_00781_asknk): Writes to non-managed Hive tables is disabled
```

sadly, we would need a separate environment with different hive.properties unlocking this... 

as this requires quite a few steps, and I am not sure of how much added value this actually brings, 
i would be OK taking a step back and not doing a test at all cc @kokosing @electrum 
(sorry, i didnt foreseen this earlier)

if we want to go with a test, i can provide necessary guidance.

Reply:
I think a test would be good and can be done if does not complicate the CI too much and is as simple as adding a new pt-config.

Or we can add a *hidden* session property (there's already more of those - like query_max_stages) to bypass for the test if we absolutely need a test.

Reply:
> ~Or we can add a _hidden_ session property (there's already more of those - like query_max_stages) to bypass for the test if we absolutely need a test.~

I guess you figured this could be a security issue.

> I think a test would be good and can be done if does not complicate the CI too much and is as simple as adding a new pt-config.

not a pt config.
rather, a new environment, similar to `singlenode` but allowing to create/write external tables
(`singlenode-with-writes-to-external` ?)

Then you'd add an invocation of `test run` to a suite 1/2/3 or 5.

Reply:
Yes, I was more hinting about cleaning up those other **hidden** session properties sometime since after discovering them I had to add controls in place to disallow setting those for non-admin users.

Thanks @findepi for guiding about the test. Since this will be a completely new test environment running essentially 1 test, would this envrionment be useful for other tests as well? Otherwise I also think it's a slight overkill for this change.